master/static/migrations/20210114153111_migrate-trial-id-to-bigint.down.sql

```diff
@@ -0,0 +1,30 @@
+DO $$
+    BEGIN
+        IF NOT EXISTS (SELECT 1
```

Comment:
I keep trying to fix the whitespace from the suggestion box but it's not getting pasted in the same 🙃 . I'll fix it before landing.